vendor/bat-native-confirmations/src/bat/confirmations/internal/confirmations_impl.cc

```diff
@@ -950,10 +950,9 @@ void ConfirmationsImpl::UpdateAdsRewards(const bool should_refresh) {
   if (!state_has_loaded_) {
     BLOG(ERROR) <<
         "Unable to update ads rewards as Confirmations state is not ready";
+    return;
   }
 
-  DCHECK(wallet_info_.IsValid());
```

Comment:
This `DCHECK` is not required as `ads_rewards_->Update(wallet_info_, should_refresh);` validates if the wallet is valid